test/e2e/disabling_copied_csv_e2e_test.go

```diff
@@ -159,8 +178,8 @@ var _ = Describe("Disabling copied CSVs", func() {
 				}
 
 				expectedCondition := metav1.Condition{
-					Reason:  "NoCopiedCSVsFound",
-					Message: "Copied CSVs are disabled and none were found for operators installed in AllNamespace mode",
+					Reason:  "CopiedCSVsDisabled",
```

Comment:
Yep, this is why I'd like to see a follow-up that moves this reason to a constant variable that can be shared around.